llvm/test/CodeGen/AMDGPU/WaveTransform/wavetransform-basic-check-rcfg.mir

```diff
@@ -0,0 +1,104 @@
+# REQUIRES: asserts
+# RUN: llc -march=amdgcn -mcpu=gfx1010 -verify-machineinstrs -amdgpu-late-wave-transform=1 -run-pass=amdgpu-wave-transform -amdgpu-wave-transform-print-final=1 -o - %S%{fs-sep}wavetransform-basic.mir 2>&1 | FileCheck -check-prefixes=CFG %s
```

Comment:
remove -verify-machineinstrs here. The original test that checks the codegen changes has the verify in it.